changelogs/fragments/9157-fix-dnf_config_manager-locale.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - dnf_config_manager - forces locale to "C" before module starts. If the locale was set to non-English, the output of the ``dnf config-manager`` couldn't be parsed. (https://github.com/ansible-collections/community.general/pull/9157, https://github.com/ansible-collections/community.general/issues/9046)
```

The formatting needs a couple of adjustments:
```suggestion
  - dnf_config_manager - forces locale to ``C`` before module starts. If the locale was set to non-English, the output of the ``dnf config-manager`` could not be parsed (https://github.com/ansible-collections/community.general/pull/9157, https://github.com/ansible-collections/community.general/issues/9046).
```